### PR TITLE
Cart functionality

### PR DESCRIPTION
**Pull request by vaishnavipaliya11**
User can add and remove Products to cart.
Based on it discount is provided. 
image

### CODE REVIEW

**Review by mohammadkhizerkhan — APPROVED** [review]

**Comment by mohammadkhizerkhan** [comment]
Instead of calling gettoken() everywhere,
You can use context for auth, and import token wherever you want.

File: src/pages/LoginPage/Login.js [hunk 1]
```diff
@@ -18,31 +18,33 @@ const Login = () => {
   const loginHandler = async (e) => {
     e.preventDefault();
     try {
-      const response = await axios.post(`/api/auth/login`, {
+      const response = await axios.post("/api/auth/login", {
         email: userDetails.email,
         password: userDetails.password,
       });
+      console.log(response);
```
Comment (shraddha15-creator) [inline src/pages/LoginPage/Login.js]: remove console

File: src/pages/LoginPage/Login.js [hunk 2]
```diff
@@ -18,31 +18,33 @@ const Login = () => {
...
       localStorage.setItem("token", response.data.encodedToken);
       setAuth(true);
       navigate("/");
       console.log(response);
```
Comment (shraddha15-creator) [inline src/pages/LoginPage/Login.js]: remove console

File: src/pages/LoginPage/Login.js [hunk 3]
```diff
@@ -18,31 +18,33 @@ const Login = () => {
...
         email: "adarshbalika@gmail.com",
-        password: "adarshBalika123",
+        password: "adarshbalika",
       });
       localStorage.setItem("token", data.encodedToken);
       setAuth(true);
       navigate("/");
     } catch (error) {
-      alert(error);
+      console.log(error);
```
Comment (shraddha15-creator) [inline src/pages/LoginPage/Login.js:44]: here you can give console.error

**Review by shraddha15-creator — APPROVED** [review]
Good folder structure 
Code is well written

**Review by JowelTisso — APPROVED** [review]
* Clean code
* Looks Good
* Remember to remove console logs

**Pull request merged**